ydb/core/mind/bscontroller/console_interaction.cpp

```diff
@@ -0,0 +1,214 @@
+#include "console_interaction.h"
+#include "impl.h"
+
+namespace NKikimr::NBsController {
+
+    void TBlobStorageController::StartConsoleInteraction() {
+        ConsoleInteraction = std::make_unique<TConsoleInteraction>(*this);
+        ConsoleInteraction->Start();
+    }
+
+    TConsoleInteraction::TConsoleInteraction(TBlobStorageController& controller)
+        : Self(controller)
+    {}
+
+    void TConsoleInteraction::Start() {
+        NTabletPipe::TClientConfig pipeConfig;
+        pipeConfig.RetryPolicy = {
+            .RetryLimitCount = 10,
+        };
+        MakeSessionBackoffTimer.Reset();
```

Comment:
don't need it; pipe backoffs automatically
just re-create pipe when receive TEvClientDestroyed, that's all
also you will always have active pipe (connected or trying to establish connection), here you have situations when you send data to the void (while backoff timer is at work)